### PR TITLE
Write vtk (if enabled) for dry runs

### PR DESCRIPTION
**Pull request by daavid00**
To write the .vtu when --enable-vtk-output=1 --enable-dry-run=1

### CODE REVIEW

**Comment by daavid00** [comment]
jenkins build this please

File: opm/simulators/flow/FlowGenericVanguard.hpp [hunk 1]
```diff
@@ -284,6 +284,8 @@ class FlowGenericVanguard {
     void updateOutputDir_(std::string outputDir,
                           bool enableEclCompatFile);
 
+    void updateNOSIM_(std::string enableDryRun);
```
Comment (akva2) [inline opm/simulators/flow/FlowGenericVanguard.hpp:287]: can be const std::string&

File: opm/simulators/flow/FlowGenericVanguard.cpp [hunk 2]
```diff
@@ -175,6 +175,34 @@ std::string FlowGenericVanguard::canonicalDeckPath(const std::string& caseName)
     throw std::invalid_argument("Cannot find input case '"+caseName+"'");
 }
 
+void FlowGenericVanguard::updateNOSIM_(const std::string_view& dryRunString)
```
Comment (akva2) [inline opm/simulators/flow/FlowGenericVanguard.cpp:178]: string_view is meant to be used as a value

File: opm/simulators/flow/FlowGenericVanguard.hpp (hunk 1 above)
Comment (daavid00) [reply opm/simulators/flow/FlowGenericVanguard.hpp]: I have rebased with std::string_view, thanks. Is there something else to be changed before calling Jenkins?

File: opm/simulators/flow/FlowGenericVanguard.cpp [hunk 3]
```diff
@@ -175,6 +175,34 @@ std::string FlowGenericVanguard::canonicalDeckPath(const std::string& caseName)
     throw std::invalid_argument("Cannot find input case '"+caseName+"'");
 }
 
+void FlowGenericVanguard::updateNOSIM_(std::string_view dryRunString)
+{
+    try {
```
Comment (akva2) [inline opm/simulators/flow/FlowGenericVanguard.cpp:180]: broken indentation. sorry for nitpicking.

**Comment by akva2** [comment]
jenkins build this please

File: opm/simulators/flow/FlowGenericVanguard.cpp (hunk 3 above)
Comment (daavid00) [reply opm/simulators/flow/FlowGenericVanguard.cpp:180]: Thanks for the review and sorry for the broken indentation.

**Pull request merged**